### PR TITLE
Replace Coinbase/Bitfinex by KYC exchange

### PR DESCRIPTION
**Pull request by yahiheb**

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
NACK for me. I still think that we should use known KYC exchanges like `Coinbase` or `Binance`

File: docs/glossary/Glossary-PrivacyWasabi.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Read more: [Use of blind signatures in CoinJoin](/using-wasabi/CoinJoin.md#zerol
 ### Cluster
 
 Which entities know about which coins.
-For example, this coin belongs to a cluster that is known by Coinbase and Alice.
+For example, this coin belongs to a cluster that is known by KYC exchange and Alice.
```
Comment (davterra) [inline docs/glossary/Glossary-PrivacyWasabi.md]:
```suggestion
For example, this coin belongs to a cluster that is known by a KYC exchange and Alice.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -311,7 +311,7 @@ Labels should contain the comma-separated names of people/entities that may be a
 
 So, a good label could be:
 
-`Alice, Bitfinex, BTCPay Server`
+`Alice, KYC exchange, BTCPay Server`
```
Comment (davterra) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
`Alice, <name of KYC exchange, if used>, BTCPay Server`
```

Comment (davterra) [reply docs/FAQ/FAQ-UseWasabi.md]: I think leaving it at just "KYC exchange" may result in some people using that label as a generic substitute for every KYC exchange they may have used. 
Each label, if a KYC exchange is involved, should name the specific exchange for better UTXO management going forward.

File: docs/glossary/Glossary-PrivacyWasabi.md (hunk 1 above)
Comment (davterra) [reply docs/glossary/Glossary-PrivacyWasabi.md]: nit

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (RiccardoMasutti) [reply docs/FAQ/FAQ-UseWasabi.md]: Exactly.

Comment (MaxHillebrand) [reply docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
`Alice, <name of KYC exchange>, BTCPay Server`
```

**Pull request merged**